modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -2,20 +2,30 @@
 //
 // * installing/installing_aws/installing-azure-customizations.adoc
 // * installing/installing_aws/installing-azure-user-infra.adoc
+// * machine_management/creating-machineset-azure.adoc
 
 ifeval::["{context}" == "installing-azure-customizations"]
 :ipi:
 endif::[]
 ifeval::["{context}" == "installing-azure-user-infra"]
 :upi:
 endif::[]
+ifeval::["{context}" == "creating-machineset-azure"]
+:mapi:
+endif::[]
 
 //mpytlak: The procedure differs depending on whether this module is used in an IPI or UPI assembly.
+//jrouth: Also some variations for when it appears in the machine management content (`mapi`).
 
 :_content-type: PROCEDURE
 [id="installation-azure-marketplace-subscribe_{context}"]
 = Selecting an Azure Marketplace image
+ifndef::mapi[]
 If you are deploying an {product-title} cluster using the Azure Marketplace offering, you must first obtain the Azure Marketplace image. The installation program uses this image to deploy worker nodes. When obtaining your image, consider the following:
+endif::mapi[]
+ifdef::mapi[]
+You can create a machine set running on Azure that deploys machines using the Azure Marketplace offering. To use this offering, you must first obtain the Azure Marketplace image. When obtaining your image, consider the following:
```

Comment:
s/using/by using/

? And if so, line 24 as well.

Reply:
actually "that use" - but very good point, it isn't clear :nerd_face: